README.md

```diff
@@ -114,6 +114,32 @@ You can quickly create a Price with the Stripe CLI like so:
 stripe prices create --unit-amount 500 --currency usd -d "product_data[name]=demo"
 ```
 
+<details>
+<summary>With Stripe Tax</summary>
+
+  Before creating a price, make sure you have Stripe Tax set up in the dashboard: [Docs - Set up Stripe Tax](https://stripe.com/docs/tax/set-up).
+
+  Because Stripe needs to know what kind of product you are selling, you need to submit a valid tax code specifying what kind of product you are selling. You can find a list of all tax codes here: [Available tax codes](https://stripe.com/docs/tax/tax-codes). In this example, we will use `txcd_99999999` which is 'General - Tangible Goods'.
+
+  ```sh
+  stripe products create \
+    --name="demo" \
+    --tax-code="txcd_99999999"
+  ```
+
+  From the response, copy the `id` and create a price. The tax behavior can be either `inclusive`, `exclusive` or `unspecified`. For our example, we are using `exclusive`.
```

Comment:
Unspecified is only for prices that don't use Stripe Tax. With Stripe Tax it has to be exclusive or inclusive. https://stripe.com/docs/tax/checkout#product-and-price-setup